.vscode/settings.json

```diff
@@ -9,5 +9,7 @@
     "editor.formatOnSave": true,
     "[typescript]": {
         "editor.defaultFormatter": "vscode.typescript-language-features"
-    }
+    },
```

Comment:
Does this file only affect folks developing mlxprs? Or is it part of the published plugin?